test/Vagrantfile

```diff
@@ -3,11 +3,9 @@ Vagrant.configure("2") do |config|
     config.vm.network "private_network", ip: "10.0.3.10"
     config.vm.box_url = "file://../packer/output-hashistack/package.box"
 
-    # Hashicorp consul ui
+    # Hashistack
```

Comment:
I think previous comment `# Hashicorp consul ui` had a better description

Reply:
This is now a part of default box